ocp_cleanup.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 source ocp_install_env.sh
 
-$GOPATH/src/github.com/openshift/installer/bin/openshift-install --log-level=debug --dir ocp destroy cluster || true
-
+sudo virsh destroy "${CLUSTER_NAME}-bootstrap"
+sudo virsh undefine "${CLUSTER_NAME}-bootstrap" --remove-all-storage
+sudo virsh vol-delete "${CLUSTER_NAME}-bootstrap.ign" --pool default
 rm -rf ocp
```

Comment:
```suggestion
rm -rf ocp
rm -rf configdrive```

Reply:
Ah I forgot about the configdrive dir, maybe we should move that under the ocp dir but for now agree lets remove it, thanks!